### PR TITLE
Create migration walkthrough

### PR DESCRIPTION
**Pull request by BHeisel**

### CODE REVIEW

**Comment by jim-phillips** [comment]
@BHeisel Please confirm all passwords/tokens, etc.  are useless, but it would be better to use generics, rather than old/outdated security data.  (e.g. YOUR_IRONMQ_TOKEN)

**Comment by BHeisel** [comment]
@jim-phillips The IronMQ credentials weren't valid. Good point though and I updated the KB

File: AFv1/migration-walkthrough.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+{
+ "VCAP_SERVICES": {
+  "ctl_mysql": [
+   {
+    "credentials": {
+     "certificate": "-----BEGIN CERTIFICATE-----\n{...}\n-----END CERTIFICATE-----",
+     "dbname": "default",
+     "host": "25dd22ae-0065-454b-8uw3-14c09814cec2.il1.dbaas.ctl.io",
+     "password": "GKQY0qbxn7i24Ftt",
+     "port": 49340,
```
Comment (jim-phillips) [inline AFv1/migration-walkthrough.md]: @BHeisel Can you change this one also?

**Comment by BHeisel** [comment]
@jim-phillips Audited all the migration KBs and changed all references to generic entries

**Pull request merged**